nft/service.go

```diff
@@ -265,14 +268,26 @@ func (s *service) minterJob(ctx context.Context, tokenID TokenID, model document
 				return
 			}
 			log.Infof("Successfully validated Proofs on cent chain for anchorID: %s", requestData.AnchorID.String())
-			// TODO: send txn to asset manager and wait for success
-			// to common.Address, tokenId *big.Int, bundleHash [32]byte, properties [][]byte, values [][]byte, salts [][32]byte, proofs [][][32]byte
-			//args = []interface{}{requestData.To, requestData.TokenID, requestData.Props, requestData.Values, requestData.Salts}
-			//mintContractABI = GenericMintMethodABI
+			if !utils.IsEmptyAddress(req.AssetManagerAddress) {
```

Comment:
Maybe a comment to explain that it will be changed to listen to bridge event as soon as the bridge is operational.